### ISSUE TITLE
NVDA/Chrome does not recognize parent or child group elements

### ISSUE THREAD

**Issue by shelbyelavsky** [fizzstudio/svg-aria-test#40]
This is an issue in reference to #24 which is related to the ["primitives and graphic-document" branch](https://github.com/fizzstudio/svg-aria-test/tree/aria_region).

<img width="1064" alt="Screenshot of NVDA in test environment. A red circle is highlighting the Text Viewer text that reads 'graphic 5 rect graphic 7 circle graphic 1 path' on one line. The next line reads ' graphic 3 text'." src="https://user-images.githubusercontent.com/98043076/155209620-a3e5ae36-8cf5-4db2-b47c-c7dffe4c023d.png">

### Issue Summary
NVDA reads the first three elements, which should be two separate child groupings, when navigating to the starting graphic element. The user is not able to drill in to navigate the other grouped elements individually (rect, circle, path). This causes the next grouping to not work properly. In direct relation, NVDA does not end the parent groups at all.

### Video of issue 
Example starts at 00:00:18
HTML example starts at 00:00:30
https://user-images.githubusercontent.com/98043076/155206883-f8b63090-dab8-43bb-a209-825eaeae131e.mp4

### Steps to reproduce
Use NVDA down arrow key once to navigate into the SVG space. There is only one step (we would expect at least ten separate elements to navigate.)

### Expected behavior (Pass/Fail)
- **FAIL** We expect that a screen reader (SR) can navigate to each element with a role and aria-label using a keyboard (SVG, g, and all SVG primitives).
- **FAIL** We expect that the visual focus indicator moves to the correct element and has the correct bounding dimensions.
- **FAIL** We expect that the visual focus indicator will remain in view at all times and may even adjust the scroll position of the page while navigating.
- **FAIL** The SVG is labelled by the chart's title and subtitle (these should be read as a single text when navigating to the root element).
- **FAIL** The title and subtitle are hidden from a screen reader because they are announced on their parent element instead. These elements should not be read out loud.
- **FAIL** The elements are grouped in pairs. The screen reader should be able to separately navigate to the group before navigating to each pair.
- **FAIL** Each primitive (rect, circle, path, text) should be accessible by a screen reader and read the number and label for that element.
- **FAIL** We expect that a SR will announce the aria-label correctly for each element, followed by its role.
- **FAIL** The aria label should announce the number, followed by the type of primitive (eg "7 Rect"), even though the text may be displayed visually in a different order. This is designed intentionally to ensure that the ARIA is working correctly.
- **FAIL** We expect that different SVG primitives (rect, circle, path, text) will be consistent with each other based on previous expectations.
- **FAIL** We expect that the SVG elements will provide the same experience as the HTML elements (below the SVG space).
- **???** We expect that if the main roles are not used, the fallback roles will be (if they are enabled). We are not sure how to design for this initially, but are mentioning it as important.

### Technical Details
NVDA 2021.3.1 (Default)
Chrome Version 97.0.4692.99
Windows 11 Build 22000.438
